caravel/forms.py

```diff
@@ -247,6 +247,11 @@ def __init__(self, viz):
                 "default": False,
                 "description": "Show the value on top of the bars or not"
             }),
+            'order_bars': (BetterBooleanField, {
+                "label": _("Sort Bars"),
+                "default": False,
+                "description": "Sort bars by x labels"
```

Comment:
`_("Sort bars by x labels"),` for translation, and  don't forget the trailing comma